recipes/types-pywin32/meta.yaml

```diff
@@ -0,0 +1,168 @@
+    - test -f $SP_DIR/win32pipe-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32pipe-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32print-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32print-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32process-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32process-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32profile-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32profile-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32ras-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32ras-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32security-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32security-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32service-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32service-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32serviceutil-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32serviceutil-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32timezone-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32timezone-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32trace-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32trace-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32transaction-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32transaction-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32ts-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32ts-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32ui-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32ui-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32uiole-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32uiole-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/win32wnet-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\win32wnet-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/winerror-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\winerror-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/winioctlcon-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\winioctlcon-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/winnt-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\winnt-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/winperf-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\winperf-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/winxpgui-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\winxpgui-stubs\\__init__.pyi exit 1  # [win]
+    - test -f $SP_DIR/winxptheme-stubs/__init__.pyi  # [unix]
+    - if not exist %SP_DIR%\\winxptheme-stubs\\__init__.pyi exit 1  # [win]
```

Comment:
B/c this is noarch there is no need to add tests for the Windows paths here. Also, I don't think we need to check for every file. Maybe just one, or just the pip check, is enough.

```suggestion
```

Reply:
The windows checks are necessary because the `test` command doesn't work during PR CI on a Windows box.

No file is privileged per se so I thought I'd include all of them, but I'll do as you suggested- one file, two commands (one for Unix, one for Windows)

Reply:
Also, @ocefpaf, any interest in being listed as a maintainer?

Reply:
I don't think pip check checks for file existence per se, so I think we would still want that.

Reply:
Ok, done and pushed.